tools/pdpd_poc/test_multiple_output.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+sys.path.append('/home/itikhonov/OpenVINO/openvino/bin/intel64/Debug/lib/python_api/python3.6/')
```

Comment:
Can you put generator script and resulting model with 'Split' to ngraph/test/files/paddlepaddle? And add it to unit tests, like it is done for others (see ngraph/test/frontend/paddlepaddle/basic_api.cpp)

Reply:
done